ha/util/health_generator/mock_health_event_publisher.py

```diff
@@ -0,0 +1,170 @@
+message bus.
+
+"""
+
+import argparse
+import sys
+import pathlib
+
+from cortx.utils.conf_store import Conf
+from ha.util.conf_store import ConftStoreSearch
+
+docker_env_file = '/.dockerenv'
+_index = 'cortx'
+cluster_config_file = '/etc/cortx/cluster.conf'
+config_file_format = 'yaml'
+
+def is_container_env() -> bool:
+    """Returns True if environment is docker container else False."""
+    docker_env_file_path = pathlib.Path(docker_env_file)
+    if docker_env_file_path.exists():
+        return True
+    return False
+
+def get_data_nodes(conf_store: ConftStoreSearch = None) -> list:
+    """
+    Fetches data node ids using HA wrapper class and displays the result.
+
+    Args:
+    conf_store: ConftStoreSearch object
+
+    Returns: list of node ids
+    """
+    data_node_ids = conf_store.get_data_pods(_index)
+    return data_node_ids
+
+def get_server_nodes(conf_store: ConftStoreSearch = None) -> list:
+    """
+    Fetches server node ids using HA wrapper class and displays the result.
+
+    Args:
+    conf_store: ConftStoreSearch object
+
+    Returns: list of node ids
+    """
+    server_node_ids = conf_store.get_server_pods(_index)
+    return server_node_ids
+
+def get_disks(args, conf_store: ConftStoreSearch = None) -> None:
+    """
+    Fetches disk ids using ConfStore search API and displays the result.
+
+    Args:
+    conf_store: ConftStoreSearch object
+    node_id: machine_id value
+    """
+    # TODO: This is temporary code till this is avalable in HA.
+    node_id = args.node_id
+    num_of_cvgs = Conf.get(_index, f'node>{node_id}>storage>num_cvg')
+    disk_ids = []
+    for cvg in range(num_of_cvgs):
+        num_of_data_disks = Conf.get(_index, f'node>{node_id}>storage>cvg[{cvg}]>devices>num_data')
+        num_of_metadata_disks = Conf.get(_index, f'node>{node_id}>storage>cvg[{cvg}]>devices>num_metadata')
+        [ disk_ids.append(Conf.get(_index, f'node>{node_id}>storage>cvg[{cvg}]>devices>data[{dt_disk}]')) for dt_disk in range(num_of_data_disks)]
+        [ disk_ids.append(Conf.get(_index, f'node>{node_id}>storage>cvg[{cvg}]>devices>metadata[{md_disk}]')) for md_disk in range(num_of_metadata_disks)]
+    print(disk_ids)
+
+def get_cvgs(args, conf_store: ConftStoreSearch = None) -> None:
+    """
+    Fetches cvg ids using ConfStore search API and displays the result.
+
+    Args:
+    conf_store: ConftStoreSearch object
+    node_id: machine_id value
+    """
+    node_id = args.node_id
+    # TODO: This is temporary code till this is avalable in HA.
+    cvg_count = Conf.get(_index, f'node>{node_id}>storage>num_cvg')
+    print([Conf.get(_index, f'node>{node_id}>storage>cvg[{cvg}]>name') for cvg in range(cvg_count)])
+
+def publish(args, conf_store: ConftStoreSearch = None) -> None:
+    """
+    publishes the message on the message bus.
+
+    Args:
+    conf_store: ConftStoreSearch object
+    node_id: machine_id value
+    config_file: config file path
```

Comment:
Docstring has undefined argument definition. Please remove them if not needed.

Reply:
Oh Yes..Updated now